### DISCUSSION TITLE
Pocketbase.dev

### DISCUSSION THREAD

**Discussion by jonathan-martz** [pocketbase/pocketbase#2644] [Show and tell]
I created a service for automatic Hosting Pocketbase.
Its currently in beta, first Version was build with a strang bash script which calls a php and now it depends on the Robo Task Runner Robo.li with Laravel which sends the Mails. As Database there is a Pocketbase Instance :D

Already tried to list it in the awesome pocketbase repo, but wasnt added because its a payed Service and not open source.

**Comment by silenceofthewans** [comment]
Cool but naming a paid service same as the open source project itself doesn't seem right and might mislead people if this gets significant SERP ranking. Naming along the lines of pockethost or pockethoster (this is available) would've been alright and much more descriptive about the purpose. Just my two cents.

**Reply by jonathan-martz** [reply]
PocketHost is 1. a open Source Project and 2. an name of a Web Hoster.

**Comment by benallfree** [comment]
> Already tried to list it in the awesome pocketbase repo, but wasnt added because its a payed Service and not open source.

Hello @jonathan-martz, I maintain [awesome-pocketbase](https://github.com/benallfree/awesome-pocketbase) and also run https://pockethost.io which is a hosting service for PocketBase too.

[As I said in the PR you submitted](https://github.com/benallfree/awesome-pocketbase/pull/19), the PR was rejected because it's not an open source project. If you release the source for your hosting platform on github, I'll happily include it. You can still run a paid service powered by the open source project, but I'd like awesome-pocketbase to remain an open source resource.

PS, I do kind of agree that `pocketbase.dev` is a little too close to PocketBase, but if @ganigeorgiev doesn't mind then 🤷  ❤️

**Reply by ganigeorgiev** [reply]
It's fine, I don't mind.
